src/js/puzzles.js

```diff
@@ -291,20 +291,20 @@ var puzzlesArray = [
   id: 42,
   sentence: 'Learn how to strive a truck.',
   word: 'strive',
-  hint: 'you need a license to do this',
+  hint: 'operate or control the truck‘s movement',
```

Comment:
Let's keep the original hint. We want to build this under the assumption non-native speakers will be playing.